app/assets/stylesheets/mobile/topic-list.scss

```diff
@@ -45,6 +50,25 @@
     position: relative;
     .navigation-toggle {
       flex: 0 1 auto;
+      border: 0;
+
+      .toggle-link {
+        font-size: var(--font-up-1-rem);
+        font-weight: bold;
+        color: var(--primary-high);
```

Comment:
What was your reason for using `primary-high` and not `primary`?

Reply:
The primary looked so harsh in bolded.